@d3fc/d3fc-element@^6.1.0 seems to be missing, breaking npm install d3fc

Hi all,

I'm not sure if this is the right place for this ticket.  I'd be happy to post somewhere else if appropriate.  I just did
```npm install d3fc```
and I get this error
```
(docker) rj-local@rj-desk2:/twbuild/firebase-nuxt/src$ npm add d3fc
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @d3fc/d3fc-element@^6.1.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'd3fc'
npm ERR! notarget 
```
I tried `d3fc@15.0.10` and that installed fine:
```
$ npm i d3fc@15.0.10 --save
...
+ d3fc@15.0.10
updated 1 package and audited 1702 packages in 5.946s
```
It seems like `@d3fc/d3fc-element@^6.0.1` exists on the NPM servers, but `@d3fc/d3fc-element@^6.1.0` does not.

Please let me know how I can help debug this. I'm just using the `15.0.10` version for now, so I'm not blocked, but it seems likely that I'm not the only person having this problem, so I wanted you to be aware.

Thanks!

--- a/packages/d3fc-element/README.md
+++ b/packages/d3fc-element/README.md
@@ -66,7 +66,6 @@ setTimeout(() => {
 }, 1000);
 ```
 
-
 <a name="surface_requestRedraw" href="#surface_requestRedraw">#</a> *surface*.**requestRedraw**()
 
 Enqueues a redraw to occur on the next animation frame, only if there isn't already one pending. If one is already pending, this call is ignored.

